src/test/resources/features/Login.feature

```diff
@@ -1,62 +1,106 @@
-    When Valid login credentials are entered
-    And 'Login' button is clicked
+
+  @deletesUserAccount
+    @positive
+  Scenario Outline: User logins with valid credentials. User is logged in
+    Given Login and Registration page is opened
+    When User is registered without login with "<valid email>" and "<valid password>"
+    And Login "<valid email>" and "<valid password>" are entered
+    And Login button is clicked
     Then User is logged in
+    Examples:
+      | valid email                    | valid password  |
+      | email@gmail.com                | validPassword1! |
+      | email1@gmail.com               | validPassword1! |
+      | email@example.com              | validPassword1! |
+      | firstname.lastname@example.com | validPassword1! |
+      | email@subdomain.example.com    | validPassword1! |
+      | firstname+lastname@example.com | validPassword1! |
+      | email@123.123.123.123          | validPassword1! |
+      | 1234567890@example.com         | validPassword1! |
+      | email@example-one.com          | validPassword1! |
+      | _______@example.com            | validPassword1! |
+      | email@example.name             | validPassword1! |
+      | email@example.museum           | validPassword1! |
+      | email@example.co.jp            | validPassword1! |
+      | firstname-lastname@example.com | validPassword1! |
+
 
   @negative
-  Scenario Outline: Login with both invalid email and password credentials
-    Given Login page is opened
-    When <invalid email> is entered
-    And <invalid password> is entered
-    And 'Login' button is clicked
-    Then Error message appears
+  Scenario: User logins with both invalid login and password credentials. Error message appears
+    Given Login and Registration page is opened
+    When Login "nonexistingemail0000@gmail.com" and "invalidPassword" are entered
+    And Login button is clicked
+    Then Login "Please enter a correct username and password. Note that both fields may be case-sensitive." appears
+
+
+  @requiresRegistrationWithoutLogIn
+    @loginsAndDeletesUserAccount
+    @negative
+  Scenario Outline: User logins with invalid email but valid password. Error message appears
+    Given Login and Registration page is opened
+    When Login "<invalid email>" and "<invalid password>" are entered
+    And Login button is clicked
+    Then Login "<error message>" appears
     Examples:
-      | invalid email          | invalid password |
-      | invalidemail@gmail.com | invalidpassword  |
-      | testemail1@gmail.com   | testpassword     |
-      | Testemail@gmail.com    | Testpassword!    |
-      | TESTEMAIL@GMAIL.COM    | TESTPASSWORD!    |
+      | invalid email                  | invalid password | error message                                                                              |
+      | EMAIL@GMAIL.COM                | VALIDPASSWORD1!  | Please enter a correct username and password. Note that both fields may be case-sensitive. |
```

Comment:
Why you consider different invalid emails as different equivalence classes? (also considering we get one same output result)